src/blog/2024-09-14/index.md

```diff
@@ -0,0 +1,73 @@
+title: Pre-Alpha v8.0 Release Note
+summary: Release note for Neotokyo; Rebuild Pre-Alpha v8.0
+author: nullsystem
+
+# Pre-Alpha v8.0 Release Note
+2024-09-14 nullsystem
+
+* [Builds download + changelog](https://github.com/NeotokyoRebuild/neo/releases/tag/v8.0-prealpha)
+
+## Highlights
+
+### Weapon spread and recoil
+Spread and recoil changes are now in-place and is parity/close to to OG:NT. There is now a difference
+between hip fire and ADS, giving a more tighter spread for ADS.
+
+### New main menu
+The main menu got an entire overhaul replacing the standard Source main menu with a custom replacement. The settings,
+server browser, new games, popups, and player lists got replaced. The work on the UI is currently on-going and while
+the server browser is missing some extras, it is usable enough to replace the old UIs currently. However, the "legacy"
+windows can be accessed in the server browser and settings by the "legacy" button on the bottom.
+
+Here's a screenshot of the settings page:
+
+[![Options menu with the new overhaul GUI](neoimgui.png_thumb.png)](neoimgui.png)
+
+For the rest, load up NT;RE Pre-Alpha v8.0 and see for yourselves.
```

Comment:
Erm, they wanna know what else is new..? Maybe just remove this entirely imo cuz it doesn't make sense.